proc/proc_linux.go

```diff
@@ -270,6 +272,10 @@ func (dbp *Process) trapWait(pid int) (*Thread, error) {
 			}
 			continue
 		}
+		if th == nil {
+			// Sometimes we get an unknown thread, ignore it?
```

Comment:
yes it was crashing, a nil pointer dereference at th.running = false later on, yes this is weird